admin-dev/themes/new-theme/js/components/multistore-header.ts

```diff
@@ -116,6 +116,50 @@ const initMultistoreHeader = () => {
 
   updateLinksAnchor();
   window.addEventListener('hashchange', updateLinksAnchor);
+
+  /**
+   * When a field from a multistore configuration form is changed,
+   * we set multistoreConfigurationFormHasChanged to true
+   */
+  const multistoreConfigurationFormContainer = document.querySelectorAll(ComponentsMap.multistoreConfigurationFormInputs);
+  let multistoreConfigurationFormHasChanged = false;
+  multistoreConfigurationFormContainer.forEach((element) => {
+    element.addEventListener('change', (event: Event) => {
+      multistoreConfigurationFormHasChanged = true;
+    });
+  });
```

Comment:
Suggestion: is it worth the effort to store the initial values in case the user reverts the changes, and cancel the modal display ?

Reply:
Earlier I suggested to store a hash of the whole form content, and check if the hash is different...

But talking with other core developers, we decided it was overkill, and accepted that if someone changes the value of a field and revert his change, he will still be notified, no big deal